src/third_party/LICENSE.md

```diff
@@ -0,0 +1,62 @@
+This license file applies to everything in this repository except that which
```

Comment:
Which file is this license used for?

Reply:
It is for the concurrentqueue, we can probably take it out or rename it.  We made a decision in RENEWlab to use the json single include file so I have updated this PR also.

Reply:
I think this license is for the whole concurrent queue library. We only take one header file from that repo and it already contains the license on the top of the file. Do we really need an extra license here?

Reply:
I agree, Removed.